src/component/header/Header.jsx

```diff
@@ -1,94 +1,85 @@
-import React, { Component } from "react"
+import React from "react"
 import { Link } from "react-router-dom"
 import { FiX, FiMenu } from "react-icons/fi"
 
-class Header extends Component {
-  constructor(props) {
-    super(props)
-    this.menuTrigger = this.menuTrigger.bind(this)
-    this.CLoseMenuTrigger = this.CLoseMenuTrigger.bind(this)
-    // this.subMetuTrigger = this.subMetuTrigger.bind(this);
-    window.addEventListener("load", function () {
-      console.log("All assets are loaded")
-    })
-  }
+const Header = (props) => {
+  window.addEventListener("load", () => {
+    console.log("All assets are loaded")
+  })
```

Comment:
I know that this was a part of the original component.

But I'm curious to see what you think these three lines are doing.

So to test you, what do you think those lines are doing?

Reply:
And I'd like to mention that you should be careful with how React renders components.

As a rule of thumb, React will run everything inside a component. So in this case, we will add a new `window load` event handler each time the header is rendered.

So to test you again, how would you approach this issue? And why? 

And are there any other ways that you can think of?

![image](https://user-images.githubusercontent.com/8443215/204079141-3170dd89-0724-447f-933e-8af38564005f.png)